sdk/keyvault/azure-security-keyvault-administration/src/test/java/com/azure/security/keyvault/administration/KeyVaultAdministrationClientTestBase.java

```diff
@@ -63,6 +74,19 @@ protected List<HttpPipelinePolicy> getPolicies() {
                 .tenantId(tenantId)
                 .additionallyAllowedTenants("*")
                 .build();
+
+            if (interceptorManager.isRecordMode()) {
+                List<TestProxySanitizer> customSanitizers = new ArrayList<>();
+                customSanitizers.add(new TestProxySanitizer("token", null, "REDACTED", TestProxySanitizerType.BODY_KEY));
+                interceptorManager.addSanitizers(customSanitizers);
+            }
+        } else {
+            credential = new MockTokenCredential();
+
+            List<TestProxyRequestMatcher> customMatchers = new ArrayList<>();
+            customMatchers.add(new BodilessMatcher());
+            customMatchers.add(new CustomMatcher().setExcludedHeaders(Collections.singletonList("Authorization")));
```

Comment:
"Authorization" was this still needed to be excluded after MockTokenCredential?

Reply:
I think I added this before moving onto `MockTokenCredential`. I can give a shot to removing it.

Reply:
Looks like we needed it :v